src/Components/Footer/Footer.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography, Grid } from '@mui/material';
+import { StyledFooter } from './StyledFooter';
+import { theme } from '../../Theme/Theme';
+import NavigationList from '../NavigationList/NavigationList';
+const categories = { listMainText: 'Shope By Category', list: ['HandBags', 'Watches', 'Skincare', 'Jewllery', 'Apparels', 'HandBags', 'Watches'] };
+const products = { listMainText: 'Shope By Product', list: ['HandBags', 'Watches', 'Skincare'] };
+import LocationOnOutlinedIcon from '@mui/icons-material/LocationOnOutlined';
+import FacebookOutlinedIcon from '@mui/icons-material/FacebookOutlined';
+import TextWithIcon from '../TextWithIcon/TextWithIcon';
+import SocialMediaLinks from '../SocialMediaLinks/SocialMediaLinks';
+
+const socialMedia = [{ Icon: FacebookOutlinedIcon, Link: 'www.facebook.com' }, { Icon: FacebookOutlinedIcon, Link: 'www.facebook.com' }, { Icon: FacebookOutlinedIcon, Link: 'www.facebook.com' }, { Icon: FacebookOutlinedIcon, Link: 'www.facebook.com' }]
```

Comment:
why using the same object (facebook) several times?

Reply:
I thought we will have these from the Backend, but I will change them